tests/test_api.py

```diff
@@ -435,6 +435,23 @@ def test_handle_error(self):
                 'foo': 'bar',
             }))
 
+    def test_errorhandler(self):
```

Comment:
per a previous comment, a test to check that overriding a default errorhandler is possible would be good to add.